frontend/src/libs/helpers/get-axis-offset/get-axis-offset.ts

```diff
@@ -0,0 +1,17 @@
+const DEEP = 0.05;
```

Comment:
I think, we could stick to files naming convention with specifying type of a code in the file.
I suggest replacing get-axis-offset.ts with get-axis-offset.helper.ts. Also, changes names on all other helpers you have created

Reply:
Yep, missed that part on file name,, helpers should be with  `.helper` suffiz

Reply:
I'm so sorry. I will make it right.
[964d9fe](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/26/commits/964d9fe50d9874b445155910c10eda371fe1bce1)